### DISCUSSION TITLE
Searching for new customtkinter libraries and widgets!

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#1684] [Showcase]
There are many **customtkinter widget libraries and tools** available on github but many of us doesn't know about them. So here is a small application I made which will help everyone explore awesome libraries.

Download the app from here: https://github.com/Akascape/tkinter-toolkit

The database can also be updated, **so anyone can share the links of libraries I missed.**

![Screenshot](https://github.com/Akascape/tkinter-toolkit/assets/89206401/fa4c6fbd-707d-468e-9f85-6c7b8ab15ceb)
![Screenshot](https://github.com/Akascape/tkinter-toolkit/assets/89206401/1884b4da-37c5-4b2c-ade4-4dd8b9012fa5)

**Comment by Dumbliidore** [comment]
You're so good, thanks.

**Comment by KotikLina** [comment]
can I translate this for my language?
